### PR TITLE
Add Images count to reply

### LINKED ISSUE #16116: Messaging - reply to message of images has wrong text
See image:

Movie_Recording_2023-05-30_17-28-47

The correct text in this example is "3 images" and it should also be shown in the messages, however it is showing "Image"

### PR DESCRIPTION
**Pull request by OmarBasem**
fixes: https://github.com/status-im/status-mobile/issues/16116

### CODE REVIEW

**Review by ibrkhalil — APPROVED** [review]

**Review by qfrank — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

File: status-go-version.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.154.5",
-    "commit-sha1": "92b5d831fe25fd2b0397519a1073a53e0be87289",
-    "src-sha256": "004pm4k090lw35kbfag0wjs815y68k61h3g6kydzpzkbmxrrfzd9"
+    "version": "2d56ddb8641e638dd0b6726e542a9333e74df71a",
```
Comment (J-Son89) [inline status-go-version.json]: this is needed?

**Review by J-Son89 — APPROVED** [review]
nice one! 👍

**Comment by VolodLytvynenko** [comment]
## ISSUE 1: Incorrect image count in reply section when replying to a message with a single image

### Steps to Reproduce:
1. Open a chat conversation that contains a message with a single image.
2. Send a reply to 1 image message
3. Observe the reply section.

### Actual Result:
Instead of displaying 'image', the reply shows '0 images' when replying to a message with a single image.
image

### Expected Result:
The reply section should correctly indicate '1 image' since the original message contains a single image.

**Comment by VolodLytvynenko** [comment]
Hi @OmarBasem 
#### One question: 
1. I noticed in the design that the term 'Photos' is used instead of 'Images' 
https://www.figma.com/file/PPWkgOYlZZDxZv5SDGsZVV/Posts-%26-Attachments-for-Mobile?type=design&node-id=2102-171079&t=H4afXlTedIE71MBb-0
2.  Additionally, if a message contains both text and images, the current text should be displayed.
image

 Could you please clarify if these changes are within the scope of the PR, or should they be addressed as a follow-up?

**Comment by OmarBasem** [comment]
Thanks @VladimrLitvinenko for your testing!

All issues are fixed!

**Comment by VolodLytvynenko** [comment]
Hi @OmarBasem unfortunately one more issue is introduced

## ISSUE 2: Quoted text extends beyond the screen in the reply section for long quoted messages with images

### Steps to Reproduce:
1. Open any chat.
2. Send a long message with images
3. Tap 'Reply'  for current message
4. Tap 'send'

### Actual result:
- The quoted text is shown beyond of the screen after step 3
- The quoted text is shown beyond of the screen after step 4 (after reply is sent)

(how it is shown on Pixel 7a)
https://github.com/status-im/status-mobile/assets/52490791/3d362af3-c2ec-4192-8b44-a6a126632f37

(how it is shown on  iPhone 11 Pro max)
image

### Expected Result:
image
https://www.figma.com/file/PPWkgOYlZZDxZv5SDGsZVV/Posts-%26-Attachments-for-Mobile?type=design&node-id=2102-171079&t=D7xAbrF5DciqV9rz-0

### Devices:
- Real device: Pixel 7a, Android 13
- Real device: iPhone 11 pro max, IOS 16

Please fix this issue to ensure that the quoted text is properly displayed within the reply section without extending beyond the screen boundaries.

**Comment by OmarBasem** [comment]
Hi @VladimrLitvinenko, the issue is fixed!

**Comment by pavloburykh** [comment]
@OmarBasem please mind my [commit](https://github.com/status-im/status-mobile/pull/16194/commits/f1d02b041e0f8d54a88673246497ca43cd01c654) when squashing and merging the PR. I have pushed small e2e fix related to PR changes. Thanx!

**Comment by VolodLytvynenko** [comment]
@OmarBasem thank you for the fixes. PR is ready to be merged

**Pull request merged**